openapi.yaml

```diff
@@ -1,279 +1,858 @@
+        - $ref: '#/components/parameters/isOpen'
+        - in: query
+          name: filter[campus][oneOf]
+          description: Name of the campus.
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - corvallis
+                - extension
+                - cascades
+                - hmsc
+                - other
+        - in: query
+          name: filter[type]
+          required: false
+          description: 'Type of location'
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - building
+                - dining
+                - cultural-center
+                - parking
+                - other
+        - in: query
+          name: filter[hasGiRestroom]
+          description: If true, locations without gender inclusive restrooms will be filtered out.
+          required: false
+          schema:
+            type: boolean
+        - in: query
+          name: filter[parkingZoneGroup][oneOf]
+          description: Locations in results should match the parkingZoneGroup in this query parameter.
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - A1
+                - A2
+                - A3
+                - B1
+                - B2
+                - B3
+                - C
+                - R
+        - in: query
+          name: filter[adaParkingSpaceCount][gte]
+          description: Only include locations with ADA parking space greater than or equal to this amount
+          required: false
+          schema:
+            type: number
+            format: integer
+        - in: query
+          name: filter[motoParkingSpaceCount][gte]
+          description: Only include locations with motorcycle parking space greater than or equal to this amount
+          required: false
+          schema:
+            type: number
+            format: integer
+        - in: query
+          name: filter[evParkingSpaceCount][gte]
+          description: Only include locations with electric vehicle parking space greater than or equal to this amount
+          required: false
+          schema:
+            type: number
+            format: integer
+        - in: query
+          name: filter[bannerAbbreviation]
+          description: Search by banner abbreviation
+          required: false
+          schema:
+            type: string
+            example: DxRC
+        - in: query
+          name: filter[arcGisAbbreviation]
+          description: Search by ArcGIS abbreviat ion
+          required: false
+          schema:
+            type: string
+            example: DxRC
+        - $ref: '#/components/parameters/pageNumber'
+        - $ref: '#/components/parameters/pageSize'
+        - in: query
+          name: include
```

Comment:
This query param needs a description. Something simple like `Include related services` should do.